tests/drone/configs/config.scality.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-return array(
-    'run'           => true,
-    'bucket'        => 'octesting',
-    'hostname'      => 'scality',
-    'port'          => '8000',
-    'key'           => 'owncloud123456',
-    'secret'        => 'secret123456',
-    'use_ssl'       => false,
-    'use_path_style'=> true,
-);
\ No newline at end of file
+return [
+	'run'           => true,
+	'bucket'        => 'owncloud',
```

Comment:
Why was this changed?

Reply:
Other apps were already using `owncloud` as the name of the bucket in tests, so that was already in the starlark, to have a bucket called `owncloud`
So it is easy to use the "standard" from other app tests.